site/installing.md

```diff
@@ -169,13 +169,16 @@ To enable DaemonSets in an existing cluster, add the `--runtime-config=extension
 
 **Install Scope on Your Cluster**
 
-It is recommended that you run Scope natively in your Kubernetes cluster using the manifest generator service.
+It is recommended that you run Scope natively in your Kubernetes cluster using `kubectl` with launch URL as show below.
 
 The simplest way to get the latest release of Scope deployed onto a Kubernetes cluster is the following:
 
-    kubectl create -f 'https://scope.weave.works/launch/k8s/weavescope.json' --validate=false
+    kubectl --namespace=kube-system create -f 'https://scope.weave.works/launch/k8s/weavescope.json' --validate=false
```

Comment:
In OpenShift you can currently do this:

```
[vagrant@localhost ~]$ oc new-project weavescope
Now using project "weavescope" on server "https://10.2.2.2:8443".

You can add applications to this project with the 'new-app' command. For example, try:

    $ oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-hello-world.git

to build a new hello-world application in Ruby.
```
In Kubernetes terms, `oc new-project` creates a namespace and switches `kubectl` context.

Next you need to do a few things to allow Scope to run:
```
[vagrant@localhost ~]$ oc adm policy add-scc-to-user privileged system:serviceaccount:weavescope:default 
[vagrant@localhost ~]$ oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:weavescope:default
```
And now you can do (note no `--validate=false` flag required actually):
```
[vagrant@localhost ~]$ oc create -f https://scope.weave.works/launch/k8s/weavescope.json
```

If we were to do what you are saying, OpenShift user would have to say something like:
```
oc create -f 'https://scope.weave.works/launch/k8s/weavescope.json?namespace=weavescope'
```